doc/mgr/dashboard.rst

```diff
@@ -314,12 +314,18 @@ The default value is 45 seconds.
 Enabling iSCSI Management
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Ceph Dashboard can manage iSCSI targets using the REST API provided
-by the `rbd-target-api` service of the `ceph-iscsi <https://github.com/ceph/ceph-iscsi>`_
-project. Please make sure that it's installed and enabled on the iSCSI gateways.
+The Ceph Dashboard can manage iSCSI targets using the REST API provided by the
+`rbd-target-api` service of the :ref:`ceph-isci`. Please make sure that it's
+installed and enabled on the iSCSI gateways.
+
+..note::
+  The iSCSI management functionality of Ceph Dashboard depends on version 3.0
```

Comment:
Unfortunately, `ceph-iscsi` 3.0 will not work well with dashboard because it doesn't include the following fix - https://github.com/ceph/ceph-iscsi/pull/52, so I'm not sure if we should wait for a 3.1 release before update the documentation.

Reply:
I'll refer to "the latest version 3 of ceph-iscsi" instead.